<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeChatBotClass</class>
 <widget class="QWidget" name="WeChatBotClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>WeChatBot</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:!hover
{ 
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(120, 120, 120, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39,39, 39, 255), stop:1 rgba(120, 120,120, 255));
border: none;
border-radius:5px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
}

QLabel
{
color: rgb(255, 255, 255);
font: 75 10pt &quot;黑体&quot;;
}

#back
{
border-radius:5px;
background-color: rgb(53, 53, 53);
}

#closeButton{
font: 150 14pt &quot;等线&quot;;
color: rgb(255, 255, 255);
background-color: rgb(53, 53, 53);
}

#title
{
color: rgb(0, 170, 255);
font: 75 12pt &quot;黑体&quot;;
}
#line
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(28, 28,28, 255), stop:1 rgba(68, 68, 68, 255));
}
</string>
  </property>
  <widget class="QWidget" name="back" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>691</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>微信机器人v1.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_Button">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QWidget" name="line" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1280</width>
      <height>2</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_Button">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QWidget" name="container" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>51</y>
      <width>590</width>
      <height>690</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#container
{
	background-color: white;
	border-radius: 5px;
}</string>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>571</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>好友</string>
     </property>
     <widget class="QListWidget" name="friend_listWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>391</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_friend_Button">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>111</width>
        <height>121</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>编辑规则</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>571</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>群聊</string>
     </property>
     <widget class="QListWidget" name="chatroom_listWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>391</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_chatroom_Button">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>111</width>
        <height>121</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>编辑规则</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="server_ip_edit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QListWidget" name="operator_list_Widget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>381</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>IP和端口</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="server_port_box">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;ADMUI3Lg&quot;;</string>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>5555</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>操作日志</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>450</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>收到数据</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="log_textEdit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>480</y>
      <width>381</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_op_Button">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>120</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_log_Button">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>450</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="wechatbot.qrc"/>
 </resources>
 <connections/>
</ui>
